.coderabbit.yml

```
@@ -1,9 +1,87 @@
+auto_assign_reviewers: true
+
+auto_review:
+enabled: true
+drafts: false
+labels:
+  - "feature"
+  - "bugfix"
+  - "!wip"
+
+path_filters:
+- "!node_modules/**"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)